lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -88,10 +92,13 @@ export class FetchClient implements INetworkModule {
                 body: reqBody,
             });
         } catch (e) {
-            throw createBrowserAuthError(
-                window.navigator.onLine
-                    ? BrowserAuthErrorCodes.postRequestFailed
-                    : BrowserAuthErrorCodes.noNetworkConnectivity
+            throw createNetworkError(
+                createBrowserAuthError(
+                    window.navigator.onLine
+                        ? BrowserAuthErrorCodes.postRequestFailed
+                        : BrowserAuthErrorCodes.noNetworkConnectivity,
+                    `error: ${e}`
```

Comment:
The second param represents subError, what you're trying to pass in here looks like additional message info

Reply:
I'd suggest adding another optional param to `NetworkError` representing the raw, underlying error

Reply:
I added it to `subError` by design since it is empty and qualifies the details. Why is another optional param preferred?

Reply:
Never mind, message is fine too. Since we do not have a class of these errors (which sub errors ideally are), your suggestion makes sense. Will make the change.

Reply:
Done.